src/core/contracts/registry.ts

```diff
@@ -101,38 +101,38 @@ export const changeHandle = async (
 };
 
 /**
- * Get all the DSNPRegistryUpdate events
- * @param filter By dsnpUserId or Contract Address
- * @param opts (optional) any config overrides.
+ * getDSNPRegistryUpdateEvents() Get all the DSNPRegistryUpdate events
+ * @param filter - By dsnpUserId or Contract Address
+ * @param opts - (optional) any config overrides.
  * @returns An array of all the matching events
  */
 export const getDSNPRegistryUpdateEvents = async (
   filter: Partial<Omit<Registration, "handle">>,
   opts?: ConfigOpts
 ): Promise<Registration[]> => {
-  const dsnpUserId = filter.dsnpUserId ? dsnpUserIdToBigNumber(filter.dsnpUserId) : undefined;
+  const dsnpUserId = filter.dsnpUserId ? convertDSNPUserIdToBigNumber(filter.dsnpUserId) : undefined;
```

Comment:
I think we can drop 'convert' here; it's common to have simply `thingoneToThingtwo` for conversions.

Reply:
We added this because we wanted all method names to start with a verb. Should we rethink that decision?

Reply:
yeah, my argument is this might be one exception to make, particularly since it's pretty standard not to have "convert" in front of conversions, especially since it makes this function name really long, OTOH if we started all conversions with "convert" then it makes it really easy to search for conversions, so 🤷‍♀️ .  It's not a blocker, just an idea.